src/main/com/smithsmodding/armory/common/logic/ArmoryInitializer.java

```diff
@@ -2,23 +2,64 @@
 
 import com.smithsmodding.armory.Armory;
 import com.smithsmodding.armory.api.common.initialization.IInitializationComponent;
-import com.smithsmodding.armory.api.util.references.References;
-import net.minecraftforge.event.RegistryEvent;
-import net.minecraftforge.fml.common.Mod;
-import net.minecraftforge.fml.common.eventhandler.SubscribeEvent;
+import net.minecraftforge.fml.common.event.*;
 
-/*
- *   ArmoryInitializer
- *   Created by: Orion
- *   Created on: 17-9-2014
- */
-@Mod.EventBusSubscriber(modid = References.General.MOD_ID)
-public class ArmoryInitializer {
+import javax.annotation.Nonnull;
+import java.util.ArrayList;
+import java.util.List;
 
-    @SubscribeEvent
-    public static void handleInitializationRegistration(RegistryEvent.Register<IInitializationComponent> initializationComponentRegisterEvent) {
-        Armory.proxy.registerInitializationComponents(initializationComponentRegisterEvent.getRegistry());
+public class ArmoryInitializer
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this class "final" or add a public constructor. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2974)